Index: src/test/resources/.~lock.TestData.xlsx#
===================================================================
--- src/test/resources/.~lock.TestData.xlsx#	(date 1608920256643)
+++ src/test/resources/.~lock.TestData.xlsx#	(date 1608920256643)
@@ -0,0 +1,1 @@
+,emresarikaya,Emre-MacBook-Pro.local,24.12.2020 16:52,file:///Users/emresarikaya/Library/Application%20Support/LibreOffice/4;
\ No newline at end of file
